libs/utils/env.py

```diff
@@ -474,19 +485,36 @@ def _init_target(self, force = False):
             # Brackets are there to let the output dir be created automatically
                                     '--workload-automation-vio={}'])
 
+            # Add variant suffix
+            suffix = os.path.splitext(os.path.basename(
+                     system_info['platform']['description']))[0]
+            board_file = self.conf['board'].lower() + suffix + '.json';
+            self._board_path = os.path.join(basepath, 'libs/utils/platforms',
+                                      board_file)
+            if not os.path.exists(self._board_path):
+                self._log.warning('Board descriptor file %s was not found.',
+                                  self._board_path)
+                self._board_path = None
+                board = None
+            else:
+                board = self._load_board()
```

Comment:
Yes, thank you, that would work nicely. For some reason I was reluctant to modify conf['board'] but looking over the code I don't see a reason not to.
@valschneider: I've updated that patch at: https://github.com/ionela-voinescu/lisa/commit/bde4b3cf0b704c4eaac80b3a04c86a0b0f64ad27.
It's being tested now but if you're okay with this version let me know and I'll merge it once I've verified it.